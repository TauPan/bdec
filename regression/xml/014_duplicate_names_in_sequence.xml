<protocol>
    <sequence name="blah">
        <sequence name="a">
            <field name="b" length="8" type="integer" />
            <field name="b" length="8" type="integer" />
        </sequence>
    </sequence>

</protocol>
