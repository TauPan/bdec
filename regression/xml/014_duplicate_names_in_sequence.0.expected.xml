
           <blah>
              <a>
                 <b>9</b>
                 <b>6</b>
              </a>
           </blah> 